cockatrice/src/player.h

```diff
@@ -4,6 +4,7 @@
 #include <QInputDialog>
 #include <QPoint>
 #include <QMap>
+#include <QRegExp>
```

Comment:
You can move this into the .cpp file, since it's not used inside the .h code.
It will avoid the inclusion of `<QRegExp>` in all the other files #including `player.h` but not using it.